### DISCUSSION TITLE
Keep getting a @babel errors!

### DISCUSSION THREAD

**Discussion by CreatiqueMedia** [sanity-io/sanity#3426] [Sanity Studio Support]
When Sanity tries to compile, I receive the following error: 

Parsing error: Cannot find module '@babel/preset-env'
Require stack:
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@babel/core/lib/config/files/plugins.js
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@babel/core/lib/config/files/index.js
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@babel/core/lib/index.js
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@babel/eslint-parser/lib/worker/babel-core.cjs
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@babel/eslint-parser/lib/worker/handle-message.cjs
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@babel/eslint-parser/lib/client.cjs
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@babel/eslint-parser/lib/index.cjs
- /Users/ron/Desktop/Dev Proj/backend_sanity/node_modules/@eslint/eslintrc/dist/eslintrc.cjs

**Comment by pran842** [comment]
Same here
